www/artists/mbolen/gallery.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="Portfolio for Connor Brezinsky">
+        <meta name="author" content="Connor Brezinsky">
+        <title>
+            Mel Bolen's Gallery
+        </title><!-- Bootstrap Core CSS -->
+        <link href="css/bootstrap.min.css" rel="stylesheet"><!-- Custom CSS -->
+        <link href="css/stylish-portfolio.css" rel="stylesheet"><!-- Custom Fonts -->
+        <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+        <link href="css/onepage-scroll.css" rel="stylesheet" type="text/css">
+		<link href="css/styles.css" rel="stylesheet">
+        <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">
+        <style type="text/css">
+        div.c9 {margin-top:10em;}
+        a.c8 { color: #050504;}
+        div.c7 {margin-top:8em;}
+        p.c6 {font-size:3em;padding-bottom:1em;}
+        p.c5 {font-family: 'Raleway', sans-serif; font-weight: 300;}
+        h1.c4 { padding-right:4em; font-family: 'Raleway', sans-serif; font-weight: 100;}
+        a.c3 { color: #ffffff;}
+        h1.c2 {font-family: 'Raleway', sans-serif; font-weight: 100;}
+        li.c1 {list-style: none; display: inline}
+        </style>
+    </head>
+</html>
```

Comment:
So, I figured out why it's not throwing an error about you not closing your head tag: you do. You just do it up here, on line 30, before you do a whole bunch of other stuff, including pretty much all of your website. You can't have anything outside of the html tag, so you need to move things into their proper place - style tag into the head, all of your sections and divs into a body tag.